.editorconfig

```diff
@@ -11,7 +11,7 @@ trim_trailing_whitespace = true
 insert_final_newline = true
 
 [*.py]
-max_line_length = 88
+max_line_length = 90
```

Comment:
Isn't 88 the black default?

Reply:
you are 100% correct. However, in edge cases Black will accept more than 88 chars, e.g. with long string constants that can not be broken up. I had to increase it to resolve such an issue with the 0001 migration. Maybe it would be even better to remove that setting and leave it to Black and Flake8 to resolve line lengths?

Reply:
The only line I see that is longer than 88 characters is https://github.com/ErikKalkoken/aa-forum/blob/improve_models_design/aa_forum/migrations/0001_initial.py#L39

And that can be simply written in two lines. Besides that one, the longest line in this migration is 87 chars.

Reply:
yes, but that is generated code, which you now have to change manually to comply with this setting. 
While both Black and Flake8 are smart enough to recognize this as acceptable exception.
So in my opinion that editorconfig checker is too rigid.

Reply:
I see your point. But I'm also not a fan of having my project settings changed without checking back with me first.

But as I said, I see your point, remove the .py setting completely than.